File: src/main/java/com/examples/testing/withemptyconstructor/correct/java/emptyconstructor/Question.java
package com.examples.testing.withemptyconstructor.correct.java.emptyconstructor;

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect;

@AlsoCorrect
public class Question {
  private int value;

  public Question() {
    value = 0;
  }

  int next() {
    return value++;
  }
}

File: src/main/java/com/examples/testing/withfeaturecheck/correct/kotlin/Question.kt
package com.examples.testing.withfeaturecheck.correct.kotlin

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect

/*
 * For testing features.
 */

@AlsoCorrect
class Question {
  fun sumTo(range: Int): Int {
    return range * (range + 1) / 2
  }
}